feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -61,8 +70,8 @@ func TestBasic(t *testing.T) {
 	isisRoot := session.ISISPath()
 	port1ISIS := isisRoot.Interface(ts.DUTPort1.Name())
 	// There might be lag between when the instance name is set and when the
-	// other parameters are set; we expect the total lag to be under 5s
-	deadline := time.Now().Add(time.Second * 5)
+	// other parameters are set; we expect the total lag to be under one minute
+	deadline := time.Now().Add(time.Minute)
```

Comment:
why one min?

Reply:
each rpc call needs 3 seconds duration and hence it is calculated as minute based on number of rpcs . At line number 133 to 147, we have 1 rpc calls in single for loop.  Hence came up with 1 minute duration for deadline variable.